### PR TITLE
Running migrations

### PR DESCRIPTION
**Pull request by gonziesc**
## Summary
- Executing migrations when they are pending to run

### CODE REVIEW

**Comment by gonziesc** [comment]
Yes! I have to change it to run automatically in production. I will assign you when it is done.

**Comment by gonziesc** [comment]
@mishuagopian @mpizzagalli Done!

File: migrations/index.js [hunk 1]
```diff
@@ -4,11 +4,14 @@ const Umzug = require('umzug'),
...
 
+const logging = migration => console.log(migration);
+
 exports.check = () => {
   const db = new Sequelize(orm.DB_URL, {
-    logging: config.environment.isDevelopment
+    logging: config.isDevelopment
   });
   const umzug = new Umzug({
+    logging,
```
Comment (mishuagopian) [inline migrations/index.js]: You can replace this with:
```js
  logging: console.log
```
And `logging` constant can be removed

File: migrations/index.js [hunk 2]
```diff
@@ -27,7 +30,13 @@ exports.check = () => {
   });
   return umzug.pending().then(migrations => {
     if (migrations.length) {
-      throw new Error('Pending migrations, run: npm run migrations');
+      if (config.isDevelopment) {
+        throw new Error('Pending migrations, run: npm run migrations');
+      } else {
+        return umzug.up().catch(err => {
+          throw new Error('There are pending migrations that could not be executed');
```
Comment (mishuagopian) [inline migrations/index.js]: Hmm, I'd also log the error (`err`) so that the user knows what's happening.

File: migrations/index.js [hunk 3]
```diff
@@ -27,7 +28,14 @@ exports.check = () => {
   });
   return umzug.pending().then(migrations => {
     if (migrations.length) {
-      throw new Error('Pending migrations, run: npm run migrations');
+      if (config.isDevelopment) {
+        throw new Error('Pending migrations, run: npm run migrations');
```
Comment (mishuagopian) [inline migrations/index.js]: Replace `Error` throwing with a rejected promise return

**Pull request merged**